addons/GearSwap/export.lua

```diff
@@ -79,34 +83,12 @@ function export_set(options)
         windower.create_dir(windower.addon_path..'data/export')
     end
     
-    local inv = items.inventory
-    if targinv then
-        -- Load the entire inventory
-        for _,v in pairs(inv) do
-            if type(v) == 'table' and v.id ~= 0 then
-                if res.items[v.id] then
-                    item_list[#item_list+1] = {}
-                    item_list[#item_list].name = res.items[v.id][language]
-                    local potslots,slot = copy_entry(res.items[v.id].slots)
-                    if potslots then
-                        slot = res.slots[potslots:it()()].english:gsub(' ','_'):lower() -- Multi-lingual support requires that we add more languages to slots.lua
-                    end
-                    item_list[#item_list].slot = slot or 'item'
-                    if not xml then
-                        local augments = extdata.decode(v).augments or {}
-                        local aug_str = ''
-                        for aug_ind,augment in pairs(augments) do
-                            if augment ~= 'none' then aug_str = aug_str.."'"..augment:gsub("'","\\'").."'," end
-                        end
-                        if string.len(aug_str) > 0 then
-                            item_list[#item_list].augments = aug_str
-                        end
-                    end
-                else
-                    msg.addon_msg(123,'You possess an item that is not in the resources yet.')
-                end
-            end
+    if all_items then
+        for i = 0, #res.bags do
+            item_list:extend(get_item_list(items[res.bags[i].en:gsub(' ', ''):lower()]))
```

Comment:
Please use `english` instead of `en`. We need to find a better way to document that, I agree, but `en` is just an implementation detail, `english` is the proper name to use.